documentation/docs/content-and-navigation/demos/skip-links.tsx

```diff
@@ -67,7 +67,7 @@ export const DemoSkipLinks = () => {
 						<a
 							className={styles.skipLinks__anchor}
 							id="website"
-							href="https://github.com/feedzai/react-a11y-tools"
+							href="https://github.com/joaotmdias/react-a11y-tools/"
```

Comment:
Just to understand, why does this href goes to your github ?

Reply:
I've commited the changes to this PR